WalletWasabi.Tests/IntegrationTests/ExtPubKeyExplorerTests.cs

```diff
@@ -107,6 +107,25 @@ public void ShouldFindUnsedKeyFarFarAway()
 			Assert.Equal(DerivateScript(true, 25_000), unusedKeyIndex);
 		}
 
+		[Fact]
+		public void DummyFilterMatchesToFalse()
+		{
+			byte[][] DummyScript  = new byte[][] { ByteHelpers.FromHex("0009BBE4C2D17185643765C265819BF5261755247D") };
+			var blockHash = RandomUtils.GetBytes(32);
```

Comment:
I do not prefer random in this kind of test. Can you make it deterministic? 
For example, get a seed and generate the first 1000 address whatever and check against them?

Reply:
Done.